### PR TITLE
README improvements and link to Glide Jobs page

### PR DESCRIPTION
**Pull request by dvdsgl**
* Improved screenshot with dark mode
* Put noisy badges below sexy screenshot
* Distill copy a bit
* Clearer link to our jobs page
* Link to Data Editor product page, not docs

CleanShot 2022-06-18 at 09 45 00

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <h1 align="center">
-  <img src="https://raw.githubusercontent.com/glideapps/glide-data-grid/master/icon.png" width="224px"/><br/>
```
Comment (jassmith) [inline README.md:2]: This breaks the icon link in nomes.com

File: README.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+  <img src="media/icon.png" width="224px"/><br/>
   <b>Glide Data Grid</b>
 </h1>
-<p align="center">A relatively small HTML5 Canvas based data editor supporting <b>millions</b> of rows, <b>rapid</b> updating, and fully <b>native scrolling</b>. We built <a href="https://grid.glideapps.com" target="_blank">Data Grid</a> as the basis for the <a href="https://docs.glideapps.com/all/reference/data-editor/introduction-to-the-data-editor" target="_blank">Glide Data Editor</a>.</p>
+<p align="center">A canvas-based data grid, supporting <b>millions</b> of rows, <b>rapid</b> updating, and <b>native scrolling</b>.</p>
+
+<p align="center">Built as the basis for the <a href="https://www.glideapps.com/data-editor" target="_blank">Glide Data Editor</a>. <a href="https://www.glideapps.com/jobs#open-positions" target="_blank">We're hiring</a>.</p>
+
+
+![Data Grid](media/data-grid.png#gh-light-mode-only)
```
Comment (jassmith) [inline README.md:10]: Not sure what this will do on npmjs.com but probably not great.

**Review by jassmith — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dvdsgl) [reply README.md:10]: Looks like this syntax is deprecated in favor of:

```html

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/25423296/163456776-7f95b81a-f1ed-45f7-b7ab-8fa810d529fa.png">
  <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
  <img alt="Shows an illustrated sun in light color mode and a moon with stars in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
</picture>
```

File: README.md (hunk 1 above)
Comment (dvdsgl) [reply README.md:2]: What is nomes.com?

**Comment by jassmith** [comment]
My phone autocorrecting npmjs.com

**Comment by dvdsgl** [comment]
Ah, will fix.

File: README.md (hunk 2 above)
Comment (dvdsgl) [reply README.md]: Ok, I am trying this syntax, but I think we have to merge to master to really know if it works

**Pull request merged**